galsim/gsobject.py

```diff
@@ -1238,10 +1239,17 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         @param surface_ops  A list of operators that can modify the photon array that will be
                             applied in order before accumulating the photons on the sensor.
                             [default: ()]
+        @param sensor       An optional Sensor instance, which will be used to accumulate the
```

Comment:
Thanks Mike,  Looks good to me.  I fixed some typos and an killed an unused variable.  Assuming that gets through travis, I think this is good to merge.